addOns/pscanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -13,6 +13,17 @@ <H2>An example passive scan rule which loads data from a file</H2>
 This implements an example passive scan rule that loads strings from a file that the user can edit.<br>
 For more details see: http://zaproxy.blogspot.co.uk/2014/04/hacking-zap-3-passive-scan-rules.html
 
+<H2>General Configuration</H2>
+
+<H3>Trusted domains</H3>
```

Comment:
Should use title caps.

https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#style-guidelines